### ISSUE TITLE
Find a better way to create / discuss / share dreamcode.

### ISSUE THREAD

**Issue by gr2m** [noBackend/nobackend.org#10]
follow up for https://gist.github.com/gr2m/5463475#comment-831841

**Comment by jessefrye** [comment]
sounds like a great idea for a hoodie app. save/vote/sync dreamcode

**Comment by gr2m** [comment]
here's my starting point: http://dreamcode.io. Wann collaborate on making it real?

**Comment by jessefrye** [comment]
yeah. i would like to help. nice meeting you in Warsaw btw. :)

**Comment by gr2m** [comment]
Wanna chat anytime soon? Are you back in Oakland? I could do tomorrow until 6pm Warsaw time

**Comment by jessefrye** [comment]
Yeah I'm back in the bay. What time zone are you in? I can sneak away from work and am around till midnight or so. my name on google is teamlinx if you want to do a hangout/calendar

**Comment by gr2m** [comment]
I'm in Zurich, 9h ahead of you. I dunno how to find you on Google (plus?), can you give me a link?

**Comment by jessefrye** [comment]
sorry got stuck at work today. i added you on g+ ill be working tomorrow on some stuff so anytime tomorrow would be good.

**Comment by gr2m** [comment]
I'm wondering if we could just use GutHub's gist API directly, as it's CORS enabled. The only thing I don't like is that it has no notifications on comments, I really hope they'll add it at some point.

​Another alternative would be to use a disqus widget below, that would also give us votes for free wich is pretty cool 

—
Gregor Martynus
